lib/model/Ingredient.dart

```diff
@@ -9,7 +9,7 @@ class Ingredient extends JsonObject {
   int rank;
   @JsonKey(includeIfNull: false)
   String id;
-  @JsonKey(nullable: false)
+  @JsonKey()
```

Comment:
Cant we just delete them if empty?

> Only required when the default behavior is not desired.

https://pub.dev/documentation/json_annotation/latest/json_annotation/JsonKey/JsonKey.html

Reply:
It seems like you are right. There is a problem though. I'm currently not able to run build_runner because `json_serializable` has not been migrated to null safety

```
[SEVERE] json_serializable:json_serializable on lib/model/Nutriments.dart (cached):

Generator cannot target libraries that have not been migrated to null-safety.
package:openfoodfacts/model/Nutriments.dart:8:7

```

Reply:
@peterwvj For the record, it's possible to run a mix of null safe/unsafe: `flutter run --no-sound-null-safety`
I don't know how this can apply to `build_runner`.